recipes/libiec61850/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy, rm, rmdir
+from os.path import join
+
+required_conan_version = ">=2.1"
+
+class Libiec61850Conan(ConanFile):
+    name = "libiec61850"
+    description = "An open-source library for the IEC 61850 protocols."
+    license = "LGPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://libiec61850.com/libiec61850"
+    topics = ("iec61850", "mms", "goose", "sampled values")
+
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    languages = ["C"]
+
+    def layout(self):
+        cmake_layout(self)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_EXAMPLES"] = False
+        tc.variables["BUILD_TESTS"] = False
+        tc.variables["BUILD_SHARED_LIBS"] = False
+        tc.variables["BUILD_STATIC_LIBS"] = True
```

Comment:
```suggestion
        tc.variables["BUILD_SHARED_LIBS"] = self.options.shared
        tc.variables["BUILD_STATIC_LIBS"] = not self.options.shared
```

But Conan is already injecting `BUILD_SHARED_LIBS`